sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_operations/_operations.py

```diff
@@ -531,8 +531,7 @@ def begin_conversation_analysis(
         Submit a collection of conversations for analysis. Specify one or more unique tasks to be
         executed.
 
-        See
-        https://learn.microsoft.com/rest/api/language/2023-04-01/conversation-analysis-runtime/submit-job
+        See https://learn.microsoft.com/rest/api/language/2023-04-01/analyze-conversation/submit-job
```

Comment:
Shouldn't these follow the same URL structure as above? I would think so since it's calculated by the publishing tool.